.github/CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# CODEOWNERS file (from GitHub template at
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+# Each line is a file pattern followed by one or more owners.
+
+################################################################################
+# These owners will be the default owners for everything in the repo. This is commented
+# out in favor of using a team as the default (see below). It is left here as a comment
+to indicate the primary expert for this code.
```

Comment:
Looks like this should still be commented out

Reply:
Good catch!!  Updated.